py_mimic_fhir/__main__.py

```diff
@@ -343,24 +358,26 @@ def validate(args, gcp_args):
         logger.error('Validation failed')
 
     # Only export if validation is successful
-    if args.export == True:
-        export_all_resources(
-            args.fhir_server, args.output_path, args.export_limit
-        )
+    ## DEPRECATED, NEED TO THINK IF THIS SHOULD BE ALLOWED ANYMORE
```

Comment:
Export parameters for gcp export are stored specifically in the export subparser. We would need to make these general arguments so both routines could access the arguments. BUT there are some export arguments that shouldn't be at the top level (I guess these could be defaulted.)

For now just don't allow export after validation. It is it's own step for now (ie you must 1. validate then 2. export